Packs/CommonScripts/Scripts/Base64EncodeV2/Base64EncodeV2_test.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+
```

Comment:
let's add test cases with foreign language as described in the issue. also geramn/spanish will be good